psh/psh.c

```diff
@@ -75,20 +76,46 @@ const psh_appentry_t *psh_findapp(char *appname) {
 }
 
 
-int psh_ttyopen(const char *dev)
+char *psh_stralloc(char *oldstr, const char *str)
```

Comment:
It's debatable if we want this as a separate function - it's used in one place only
```suggestion
static char *psh_stralloc(char *oldstr, const char *str)
```

Reply:
Done.